content/zh/docs/tasks/extend-kubernetes/setup-konnectivity.md

```diff
@@ -32,15 +32,8 @@ You need to configure the API Server to use the Konnectivity service
 and direct the network traffic to the cluster nodes:
 
 1. Make sure that
-the `ServiceAccountTokenVolumeProjection` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/)
```

Comment:
Please run the following command in your local clone, under the **main** branch:

```
./scripts/lsync.sh content/zh/docs/tasks/extend-kubernetes/setup-konnectivity.md
```

You will see that line 16 in the English content has changed and a paragraph has been added.

Reply:
@tengqm  this command is very useful, thanks for your advice.  PR update.